streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamFlatMapValuesTest.java

```diff
@@ -42,7 +43,7 @@ public void testFlatMapValues() {
                 @Override
                 public Iterable<String> apply(String value) {
                     ArrayList<String> result = new ArrayList<String>();
-                    result.add(value.toLowerCase());
+                    result.add(value.toLowerCase(Locale.getDefault()));
```

Comment:
Not sure about the streams tests using the default locale. Won't this cause the tests to potentially fail depending on the locale?